AsyncSwift/Views/EventView.swift

```diff
@@ -5,16 +5,267 @@
+            var name: String
+            var imageURL: String
+            var role: String
+            var description: String
+        }
+
+        struct Paragraph: Hashable {
+            var content: String
+        }
+    }
+}
+
+
 struct EventView: View {
+
+    private let data = Mock.data
+
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        NavigationView {
+            ScrollView {
+                Header
+                LazyVStack {
+                    ForEach(data.sessions) { session in
+                        makeSessionCell(for: session)
+                    }
+                }
+            }
+            .navigationTitle(Tab.event.title)
+        }
     }
 }
 
-struct ScheduleView_Previews: PreviewProvider {
-    static var previews: some View {
-        EventView()
+private extension EventView {
+
+    var Header: some View {
+        VStack(alignment: .leading, spacing: 8) {
+            Text(data.event.subject)
+                .font(.system(size: 28, weight: .bold))
+            HStack {
+                Text(data.event.title)
+                    .font(.caption2)
+                    .fontWeight(.bold)
+                    .foregroundColor(.white)
+                    .padding(.vertical, 4)
+                    .padding(.horizontal, 8)
+                    .background(Color.seminarOrange)
+                    .cornerRadius(20)
+                Spacer()
+            }
+
+            NavigationLink {
+                EventDetailView(event: data.event)
+            } label: {
+                Text("세미나 살펴보기 \(Image(systemName: "arrow.right"))")
+                    .font(.system(size: 13))
+                    .fontWeight(.bold)
+            }
+        }
+        .padding(.horizontal, 16)
+        .padding(.vertical, 30)
     }
+
+    @ViewBuilder
+    func makeSessionCell(for session: EventModel.Session) -> some View {
+        NavigationLink {
+            SessionView(session: session)
+        } label: {
+            VStack {
+                CustomDivider()
+                HStack {
+                    VStack(alignment: .leading, spacing: 2) {
+                        Text(session.title)
+                            .font(.headline)
+                            .foregroundColor(.black)
+                            .multilineTextAlignment(.leading)
+                        Text("\(session.speaker.name) 님")
+                            .font(.body)
+                            .foregroundColor(.black)
+                    }
+                    .padding(.vertical, 30)
+                    Spacer()
+                    VStack {
+                        Spacer()
+                        Image(systemName: "chevron.right")
+                            .font(Font.system(size: 30, weight: .light))
+                            .foregroundColor(.black)
+                        Spacer()
+                    }
+                }
+                .padding(.horizontal, 16)
```

Comment:
defualt padding 값이 16이라 안적는게 좋을듯합니다! ios개발자라면 이정도는 알아야죠!

```suggestion
                .padding(.horizontal)
```

Reply:
수정 확인했습니다